### ISSUE TITLE
pre-trained SIF!

### ISSUE THREAD

**Issue by nstfk** [PrincetonML/SIF#32]
Are there any pre-trained SIF available to be used out of the box ?

**Comment by damienlancry** [comment]
I dont think so although you only need a pre trained word embedding and a dictionnary of unigram probabilities to make your own SIF. also the PCR is supposed to be to be particular of your corpus but i guess you could just compute the first singular vector of a generic corpus like wikipedia or common crawl
